prow/integ-suite-kind.sh

```diff
@@ -157,7 +158,7 @@ export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
 trace "init" make init
 
 if [[ -z "${SKIP_SETUP:-}" ]]; then
-  export DEFAULT_CLUSTER_YAML="./prow/config/default.yaml"
+  export DEFAULT_CLUSTER_YAML="${ROOT}/${CLUSTER_YAML}"
```

Comment:
What if `CLUSTER_YAML` is an absolute path?

Reply:
what's the user case?
this's a new environment flag, and I only want it be able to use something not default.

Reply:
I just feel like our paths should be able to handle both relative and absolute paths. Regardless, if you want to use a config that's not the default, you can set the `CONFIG` environment variable:

https://github.com/istio/istio/blob/2f99077494002e6b61e6eefa169ac12ae5bbfde0/common/scripts/kind_provisioner.sh#L175-L178